cpp/test/random/make_regression.cu

```diff
@@ -161,4 +163,124 @@ TEST_P(MakeRegressionTestD, Result)
 }
 INSTANTIATE_TEST_CASE_P(MakeRegressionTests, MakeRegressionTestD, ::testing::ValuesIn(inputsd_t));
 
+template <typename T>
+class MakeRegressionMdspanTest : public ::testing::TestWithParam<MakeRegressionInputs<T>> {
+ public:
+  MakeRegressionMdspanTest() = default;
+
+ protected:
+  void SetUp() override
+  {
+    // Noise must be zero to compare the actual and expected values
+    T noise = (T)0.0, tail_strength = (T)0.5;
+
+    rmm::device_uvector<T> data(params.n_samples * params.n_features, stream);
+    rmm::device_uvector<T> values_cm(params.n_samples * params.n_targets, stream);
+    rmm::device_uvector<T> coef(params.n_features * params.n_targets, stream);
+
+    using index_type = typename rmm::device_uvector<T>::index_type;
+    using matrix_view =
+      raft::device_matrix_view<T, raft::matrix_extent<index_type>, raft::row_major>;
+    matrix_view out_mat(data.data(), params.n_samples, params.n_features);
+    matrix_view values_mat(values_ret.data(), params.n_samples, params.n_targets);
+    matrix_view coef_mat(coef.data(), params.n_features, params.n_targets);
+
+    // Create the regression problem
+    make_regression(handle,
+                    out_mat,
+                    values_mat,
+                    params.n_informative,
+                    stream,
+                    coef_mat,
+                    params.bias,
+                    params.effective_rank,
+                    tail_strength,
+                    noise,
+                    params.shuffle,
+                    params.seed,
+                    params.gtype);
+
+    // FIXME (mfh 2022/09/07) This test passes even if I don't call make_regression.
```

Comment:
Can we make a github issue for this and reference it here just to keep it on the radar?

Reply:
@cjnolet Done! https://github.com/rapidsai/raft/issues/814